### ISSUE TITLE
PANW module - Filebeat - Cortex Data Lake

### ISSUE THREAD

**Issue by Sho-Nuff-007** [elastic/beats#24165]
The enhancement would allow a user to send logs that have been downloaded from Cortex Data Lake which sends out logs that look slightly different than those seen here: https://github.com/elastic/beats/blob/master/x-pack/filebeat/module/panw/panos/test/pan_inc_traffic.log

My use case would be to receive the files via syslog (only way to get them as of today from Cortex Data Lake) which I then store locally. Filebeat would utilize its "panw" module to grab the events and forward to Elasticsearch.

 
Today, these events will still be forwarded to elasticsearch/Logstash/Kafka but the issue is that NONE of the firewall logs fields get parsed. In almost every log I see from Cortex Data lake there are many missing fields so not sure if that is the reason or just the general flow of the log file.

I have already started a case with support and they have assured me that my filebeat is setup correctly and I have also tried sending logs BOTH direct to elastic and through Kafka/Logstash and both results are the same (fields don't get parsed). I think that the 1st field for sure is different but the rest (or most) should be the same.

**Comment by andrewkroh** [comment]
> Cortex Data Lake which sends out logs that look slightly different

How do they differ? Can you provide a raw message example?

**Comment by TheChedda** [comment]
Sure thing. I have sanitized the below log but let me know if you have any questions on it. This is just for TRAFFIC but I can post a THREAT log tomorrow if needed.

`Traffic:
Feb 22 19:38:23 34.67.106.77 670 <14>1 2021-02-23T03:38:23.095Z stream-logfwd20-807132656-02191113-xjf2-harness-jxzn logforwarder - panwlogs - 2021-02-23T03:38:22.000000Z,no-serial,TRAFFIC,end,9.1,2021-02-23T03:38:12.000000Z,<Internal_IP>,<Public_IP>,<Public_IP>,<Public_IP>,My-Rule,,,ssl,vsys1,source_zone,untrust,tunnel.101,ethernet1/1,CDL_Logging,1436680,1,64555,8080,21262,8080,tcp,allow,10134,3114,7020,24,2021-02-23T03:35:55.000000Z,122,Custom-microsoft,4967399,10.0.0.0-10.255.255.255,US,11,13,tcp-fin,25,0,0,0,,REMOTE_FIREWALL,from-policy,,,0,,0,1970-01-01T00:00:00.000000Z,N/A,0,0,0,0,19e929b4-2474-4a75-8ebb-1957c5e95653,0,0,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,`

[Traffic_Log.txt](https://github.com/elastic/beats/files/6026664/Traffic_Log.txt)

I attached the log in a text file along with that I believe the fields are supposed to represent. I could be off on 1 or 2 as I did mostly a best guess on what the fields should likely be.
